CMakeLists.txt

```
@@ -235,11 +235,11 @@ if(NOT OPENMP_RUNTIME STREQUAL "NONE")
       set(CMAKE_EXE_LINKER_FLAGS "${CMAKE_EXE_LINKER_FLAGS} /nodefaultlib:vcomp")
       set(CMAKE_SHARED_LINKER_FLAGS "${CMAKE_SHARED_LINKER_FLAGS} /nodefaultlib:vcomp")
     endif()
-  elseif(OPENMP_RUNTIME STREQUAL "COMP")
+  elseif(OPENMP_RUNTIME STREQUAL "COMP)
```

```suggestion
  elseif(OPENMP_RUNTIME STREQUAL "COMP")
```

Almost there. You accidentally removed a quote.

I will approve the change after this fix.